python/langsmith/_testing.py

```diff
@@ -188,7 +211,7 @@ def wrapper(*test_args, **test_kwargs):
 
 def _get_experiment_name() -> str:
     # TODO Make more easily configurable
-    prefix = ls_utils.get_tracer_project(False) or "TestSuite"
+    prefix = ls_utils.get_tracer_project(False) or "TestSuiteResult"
```

Comment:
Think naming makes more sense as a "result" for the experiment and "Test Suite" for the dataset it creates